source/android/adaptivecards/build.gradle

```diff
@@ -42,6 +42,40 @@ task sourceJar(type: Jar) {
     classifier "sources"
 }
 
+task createPom {
+    doLast {
+        pom {
+            project {
+                groupId 'io.adaptivecards'
+                artifactId 'adaptivecards-android'
+                version System.getenv('ADAPTIVECARDSVERSION') ?: '1.1.0'
```

Comment:
I'd rather if we have an invalid version such as 0.0.0 if the setup is wrong then to potentially create a package that could be interpreted as valid. We now have 2 places that need to be updated when we bump up the version.